src/Poseidon/CLI/FStats.hs

```diff
@@ -216,8 +218,10 @@ getPopIndices indEntries popSpec =
 
 -- | The main function running the FStats command.
 runFstats :: FstatsOptions -> IO ()
-runFstats (FstatsOptions baseDirs jackknifeMode exclusionList statSpecsDirect maybeStatSpecsFile rawOutput) = do
-    packages <- loadPoseidonPackages baseDirs
+runFstats (FstatsOptions baseDirs jackknifeMode exclusionList statSpecsDirect maybeStatSpecsFile rawOutput ignoreChecksums) = do
+    packages <- if ignoreChecksums 
+                then loadPoseidonPackagesIgnoreChecksums baseDirs
```

Comment:
I think it would be better if `ignoreChecksums` would be a simple argument to `loadPoseidonPackages`... that would already save one `load...` function